app/src/main/java/org/kiwix/kiwixmobile/settings/CustomSwitchPreference.java

```diff
@@ -64,6 +69,19 @@ protected void onBindView(View view) {
     ViewGroup viewGroup = (ViewGroup) view;
     clearListenerInViewGroup(viewGroup);
     super.onBindView(view);
+    setGrayState(view);
+  }
+
+  private void setGrayState(View view) {
```

Comment:
Have you reformatted the code using square codestyle?

Reply:
What's the issue?

Reply:
There will be code indent and other formatting issues if you haven't used square.

Reply:
I know but is there any issue?